shell/platform/fuchsia/flutter/session_connection.cc

```diff
@@ -76,18 +88,18 @@ FlutterFrameTimes DefaultSessionConnection::GetTargetTimes(
   // Useful knowledge for analyzing traces.
   fml::TimePoint previous_vsync = next_vsync - vsync_interval;
   TRACE_DURATION(
-      "flutter", "DefaultSessionConnection::GetTargetTimes",
-      "previous_vsync(ms)", previous_vsync.ToEpochDelta().ToMilliseconds(),
-      "last_targeted(ms)", last_targeted_vsync.ToEpochDelta().ToMilliseconds(),
-      "now(ms)", fml::TimePoint::Now().ToEpochDelta().ToMilliseconds(),
-      "next_vsync(ms))", next_vsync.ToEpochDelta().ToMilliseconds(),
-      "frame_start_time(ms)", frame_start_time.ToEpochDelta().ToMilliseconds(),
-      "frame_end_time(ms)", frame_end_time.ToEpochDelta().ToMilliseconds());
+      "flutter", "SessionConnection::GetTargetTimes", "previous_vsync(ms)",
```

Comment:
Because this diff is hard to read:

I changed the fml::TimePoint::Now() call to use the local `now` variable instead.